packages/ckeditor5-ui/src/menubar/utils.ts

```diff
@@ -182,6 +186,9 @@ export const MenuBarBehaviors = {
 			}
 		} );
 
+		// After clicking menu bar list item the focus is moved to the newly opened submenu.
+		// We need to enable focus border for the submenu items because after pressing arrow down it will
+		// focus second item instead of first which is not super intuitive.
```

Comment:
After adding this comment I see your point now, and I think I agree with it. The crucial thing here is that we change the focus but this is not visualized and the blue border is the best way to visualize it.

Reply:
Yep, exactly.